go.mod

```diff
@@ -132,6 +147,6 @@ require (
 replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.25
+	github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.26
```

Comment:
Here they say the cometbft repo can be used:
https://docs.cometbft.com/v0.34/introduction/upgrading-from-tm

Reply:
Okay, the switch to cometbft comes with sdk 0.45.15:
https://github.com/cosmos/cosmos-sdk/compare/v0.45.14...v0.45.15-rc.0

Reply:
yes I think for v0.45 will be a simple replace